src/fmu/dataio/_model/fields.py

```diff
@@ -556,6 +557,16 @@ class Context(BaseModel):
     See :class:`enums.FMUContext`."""
 
 
+class CaseContext(Context):
+    stage: Literal[enums.FMUContext.case] = Field(default=enums.FMUContext.case)
+
+
+class IterationContext(Context):
+    stage: Literal[enums.FMUContext.iteration] = Field(
+        default=enums.FMUContext.iteration
+    )
```

Comment:
Can you doc string these, even if they are sort-of self-descriptive? It will end up built into the documentation like https://fmu-dataio.readthedocs.io/en/latest/datamodel/model/ObjectMetadata/FMU/fmu.dataio._model.fields.Context.html

Reply:
Doc strings added.